ts/utils/staking_utils.ts

```diff
@@ -0,0 +1,55 @@
+import * as _ from 'lodash';
+
+import { PoolWithStats, StakingPoolReccomendation } from '../types';
+
+interface PoolStatSummary {
+    poolId: string;
+    operatorShare: number;
+    zrxStaked: number;
+    sevenDayProtocolFeesGeneratedInEth: number;
+}
+
+interface GetRecommendedStakingPoolsOptions {
+    alpha: number;
+    numIterations: number;
+}
+
+export const stakingUtils = {
+    getRecommendedStakingPools: (amountZrxToStake: number, pools: PoolWithStats[], opts?: Partial<GetRecommendedStakingPoolsOptions>): StakingPoolReccomendation[] => {
+        if (!pools || amountZrxToStake === 0) {
+            return [];
+        }
+        const { alpha, numIterations } = {
+            alpha: 2 / 3,
```

Comment:
alpha needs to be obtained from the DB or Web3

Reply:
Let me see if that is possible. Otherwise I don't expect this value to change very often correct?

Reply:
I spoke to greg. I don't think we'll be changing this very often so not worth fetching. Can always update the frontend.